modules/appeals_api/spec/support/shared_examples_contestable_issues.rb

```diff
@@ -138,8 +138,13 @@ def get_issues(options:, ssn: '872958715', file_number: nil, receipt_date: '2019
     end
 
     path = if options[:version] == 'v2'
-             "/services/appeals/v2/decision_reviews/contestable_issues/#{options[:decision_review_type]}" \
-               "?benefit_type=#{options[:benefit_type]}"
+             (
```

Comment:
In this case I just went ahead and added another option to the existing shared examples... it occurs to me now that we could do something like this with the legacy appeals tests (i.e. break out a set of shared examples to run for both the original endpoint and the new one) instead of what I did above - that would take a bit longer, but lmk if it sounds helpful.